package.json

```diff
@@ -41,33 +41,33 @@
   "requiredAngularVersion": "^8.0.0 || ^9.0.0-0",
   "requiredMDCVersion": "^4.0.0-alpha.0",
   "dependencies": {
-    "@angular/animations": "^9.0.0-next.10",
-    "@angular/common": "^9.0.0-next.10",
-    "@angular/compiler": "^9.0.0-next.10",
-    "@angular/core": "^9.0.0-next.10",
-    "@angular/elements": "^9.0.0-next.10",
-    "@angular/forms": "^9.0.0-next.10",
-    "@angular/platform-browser": "^9.0.0-next.10",
+    "@angular/animations": "9.0.0-next.10",
+    "@angular/common": "9.0.0-next.10",
+    "@angular/compiler": "9.0.0-next.10",
+    "@angular/core": "9.0.0-next.10",
+    "@angular/elements": "9.0.0-next.10",
+    "@angular/forms": "9.0.0-next.10",
+    "@angular/platform-browser": "9.0.0-next.10",
```

Comment:
Might as well bump to next.11 which got release yesterday.

Reply:
As mentioned in the OP, Angular `9.0.0-next.11` breaks "everything", i.e. hundreds of type checking errors, even with `"fullTemplateTypeCheck": false`. I've raised the issue with Joost and Jeremy has discussed it with Kara and Alex R. Hopefully `next.12` will be better?